requirements.txt

```diff
@@ -36,7 +36,7 @@ geoalchemy2==0.9.4
     # via -r requirements.in
 geojson==2.5.0
     # via -r requirements.in
-greenlet==1.1.3
+greenlet==3.0.3
```

Comment:
- fixer la version de greenlet sera à faire dans `requirements.in`,
- pas pu tester sur Debian 11 pour le moment,
- cependant je ne vois pas d'utilisation de SQLAlchemy dans un mode asynchrone dans GN-Atlas, je ne pense pas que `greenlet` soit utilisé, la dépendance est juste incluse par défaut,
- ce serait intéressant d'avoir l'erreur rencontrée lors de l'installation sur Ubuntu 24.04 pour documenter le pourquoi de cette montée de version.

Reply:
Oui en effet. 
Concernant `greenlet` voir aussi les discussions : 
- https://github.com/PnX-SI/GeoNature-atlas/issues/494
- https://github.com/PnX-SI/GeoNature-atlas/issues/501

Reply:
Pour cela, j'ai rajouté un test dans les github actions pour tester l'installation des requirements.txt sur les différentes versions de python.